salesforce/my-conservation-life/force-app/main/default/lwc/geoQuery/geoQuery.js

```diff
@@ -1,26 +1,27 @@
+/* eslint-disable vars-on-top */
 /* eslint-disable no-console */
 import { LightningElement } from 'lwc';
 
+import leafletDraw from '@salesforce/resourceUrl/leafletDraw';
+
 import utils from 'c/utils';
 
 const DISTANCE_URL = utils.URL + 'assets/geometrySearch/distance';
+const POLY_URL = utils.URL + '/assets/geometrySearch/polygon';
```

Comment:
Either remove the first '/' or add it to the above line for consistency

Reply:
Resolved